src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.cs

```diff
@@ -58,7 +58,7 @@ public void TestApplicationGatewayCRUD2()
             TestRunner.RunTestScript(string.Format("Test-ApplicationGatewayCRUD2 -baseDir '{0}'", AppDomain.CurrentDomain.BaseDirectory));
         }
 
-        [Fact]
+        [Fact(Skip="Disabled due to consistent InternalServerError")]
```

Comment:
@number213 , why the test is disabled?

Reply:
@erich-wang there were internal errors that AppGw team was looking at, so I disable test temporary to get CI to pass. We found a way to make them work, so they are unskipped now